modules/gapi/include/opencv2/gapi/s11n.hpp

```diff
@@ -173,6 +273,7 @@ IOStream& operator<< (IOStream& os, const std::map<K, V> &m) {
     for (const auto& it : m) os << it.first << it.second;
     return os;
 }
+
```

Comment:
You shouldn't do this, please keep it like it was before.

The grouping allows to easily match `<<` with `>>`.

Reply:
Done